kolibri/plugins/learn/assets/src/vue/explore-page/index.vue

```diff
@@ -69,8 +91,133 @@
+    .tool-bar-container .search-input-active
+      width: $thumbnail-width * (n-cols) * 0.7
+    .tool-bar-container .tool-bar-center
+      -webkit-transform: translateX(-(((($thumbnail-width * n-cols) + $margin-width * (n-cols - 1)) - ($thumbnail-width * (n-cols) * 0.7))/2)px)
+  .section-title
+    margin-top: 5vh
+    font-size: 1.2em
+    font-weight: 700
+  .tool-bar-container
+    position: fixed
+    top: 0
+    card-list-container-width(1)
+    height: 30px
+    padding: 30 0
+    background-color: $core-bg-canvas
+    text-align: center
+    z-index: 1
+    .breadcrumbs-container
+      position: absolute
+      left:0
+      display: inline-block
+      line-height: 30px
+      z-index: -1000
+    .tool-bar
+      float: right
+      display: inline-block
+      opacity: 0.6
+      transition: all 0.3s ease-out
+    .tool-bar-center
+      height: 30px
+      text-align: center
+    .tool-bar-center .btn-search
+      pointer-events: none
+      top: 0
+    .searchform
+      display: inline-block
+      z-index: 1
+    .search-input
+      position: relative
+      display: inline-block
+      width:0
+      background-color: $core-bg-canvas
+      border-radius: 40px
+      border: 1px solid rgba(58, 58, 58, 0.1)
+      border:none
+      pointer-events: none
+      transition: width 0.2s ease-out
+      -webkit-backface-visibility: hidden
+      -webkit-transform: translate3d(0, 0, 0)
+    .search-input-active
+      input-width(1)
+      display: inline-block
+      padding: 0 40px
+      height:30px
+      border: 1px solid rgba(58, 58, 58, 1)
+      pointer-events: auto
+    .btn-search-left
+      left: 40px
+    .btn-search
+      position: relative
+      display: inline-block
+      height: 30px
+      width: 30px
+      background:none
+      border: none
+      text-indent: -10000px
+      cursor: pointer
+      z-index: 1
+      top: 4
+  .close
+    position: relative
+    display: inline-block
+    right: 40px
+    height: 30px
+    width: 30px
+    background:none
+    border: none
+    text-indent: -10000px
+    cursor: pointer
+    z-index: 1
+  .card-section
+    position: relative
+    top: 60px
+  .card-list-container
+    card-list-container-max-width(3)
+    margin-right: -($margin-width)
+  .card-list
+    margin: auto
+  .card-list .card
+    margin: 0 $margin-width $margin-width 0
+  .page
+    margin-left: 80px
+  .page-container
+    max-width: (200*3) + 24*2
+    margin:auto
+  .fast-transition
+    transition: all 0.3s ease-out
+  .fast-enter
+    opacity: 0
+    transform: translateX(-50%)
+  .fast-leave
+    opacity: 0
+    transform: translateX(-100%)
+  @media screen and (min-width: 570px)
+    media-query(2)
+  @media screen and (min-width: 810px)
+    media-query(3)
+  @media screen and (min-width: 1060px)
+    media-query(4)
+  @media screen and (min-width: 1280px)
+    media-query(5)
+  @media screen and (min-width: 1680px)
+    media-query(6)
```

Comment:
Looks like a lot of the code from the search widget is accidentally duplicated here in `explore-page/index.vue`. Would you mind reverting this file back to its original state, and then adding the search widget with the minimal changes possible?

thank you!